app/javascript/components/timeline-options/timeline-options.jsx

```diff
@@ -88,11 +86,11 @@ const TimelineOptions = ({ url }) => {
 };
 
 TimelineOptions.propTypes = {
-  url: PropTypes.string,
+  submitChosenFormOptions: PropTypes.func.isRequired,
```

Comment:
we can remove `isRequired` for `submitChosenFormOptions` since we have `defaultProps`